README.md

```diff
@@ -790,6 +790,14 @@ not to e.g. an experimental tool or long-term project. To find out the current L
 use a tool such as [nvm](https://github.com/nvm-sh/nvm) or simply check the Node.js
 [website](https://nodejs.org/en/download).
 
+### Package Manager
```

Comment:
Although I don't like pinning versions in general docs, should we at least specify to use npm v7+ as a *minimum*, which comes bundled with node 16, or should be installed manually?

Reply:
> Being shipped together with Node.js also means that developers have to worry less about their package manager, as long as their Node.js version is up to date with out recommendations.

i.e. you shouldn't worry about your npm version as Node.js upgrades take care of it.

Reply:
Unless for some reason you (need to) use node 14

Reply:
If we change (in the section above)

> You **must** always use the **LTS**

to

> You **must** always use the **Active LTS**

which is node 16 currently, then the

> as long as their Node.js version is up to date with out recommendations

should be sufficient for the current and future state.